src/OpenTelemetry.Exporter.Console/README.md

```diff
@@ -1,21 +1,7 @@
 # OpenTelemetry.Exporter.Console
 
-This is a simple exporter that that JSON serializes collected spans and prints them to the Console and is intended to be used during learning how spans are creating and exported.
+This is a simple exporter that that JSON serializes collected activities and prints them to the Console and is intended to be used during learning how activities are created and exported.

```

Comment:
Not blocking this PR, should we use "activities" or "[Activity](https://docs.microsoft.com/dotnet/api/system.diagnostics.activity) objects" across all the comments and documents?

Reply:
Agree we need a consistent way.

Something like this

docs:
[Activity](https://docs.microsoft.com/dotnet/api/system.diagnostics.activity) objects (Activity is the .NET implementation of OpenTelemetry [Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span))

Code comments:
Activity objects.